### PR TITLE
Model loader changed and hats folder moved to the models folder

### LINKED ISSUE #56: Use Forge's SeparatePerspectiveModel
Use Forge's  [SeparatePerspectiveModel](https://github.com/MinecraftForge/MinecraftForge/blob/1.16.x/src/main/java/net/minecraftforge/client/model/SeparatePerspectiveModel.java) instead of our `PerspectiveAwareModel`.

This essentially involves finding all of our JSON models that use `craftingdead:perspective_aware` as their model loader (things like hats and masks) and change them to use `forge:separate-perspective`. Look at `SeparatePerspectiveModel` to see how to define the different perspective models as it's different to our `PerspectiveAwareModel`.

### PR DESCRIPTION
**Pull request by MrTommyt**
The model loader has changed from _forge:perspective_aware_ to _forge:separate-perspective_ and the hats folder has moved from _assets/craftingdead/models/item_ to _assets/craftingdead/models_

### CODE REVIEW

**Comment by josephtarbit** [comment]
Looks good. Only thing is, the forge seperate perspective model loader is already registered by Forge itself so you don't need to register it in our mod. So could you remove that registration code and then bulk find and replace all the models that reference `craftingdead:separate-perspective` and change them to `forge:separate-perspective`.

**Comment by josephtarbit** [comment]
And while you're at it, you can remove our perspective aware model loader too.

**Comment by MrTommyt** [comment]
Anything else uses that model loader, right?

**Comment by josephtarbit** [comment]
> Anything else uses that model loader, right?

Not to my knowledge.

**Comment by MrTommyt** [comment]
After tons of commits. Finally. A good rebase.

File: crafting-dead-core/src/main/java/com/craftingdead/core/client/ClientDist.java [hunk 1]
```diff
@@ -106,20 +100,9 @@
...
-import net.minecraftforge.client.event.FOVUpdateEvent;
-import net.minecraftforge.client.event.GuiOpenEvent;
-import net.minecraftforge.client.event.InputEvent;
-import net.minecraftforge.client.event.ModelBakeEvent;
-import net.minecraftforge.client.event.ModelRegistryEvent;
-import net.minecraftforge.client.event.ParticleFactoryRegisterEvent;
-import net.minecraftforge.client.event.RenderGameOverlayEvent;
-import net.minecraftforge.client.event.RenderLivingEvent;
-import net.minecraftforge.client.event.TextureStitchEvent;
+import net.minecraftforge.client.event.*;
```
Comment (josephtarbit) [inline crafting-dead-core/src/main/java/com/craftingdead/core/client/ClientDist.java]: Can you remove this wildcard import and restore the individual event imports - I assume this was your IDE's formatter so you might wanna change the settings on that.

Comment (MrTommyt) [reply crafting-dead-core/src/main/java/com/craftingdead/core/client/ClientDist.java]: Done!

**Pull request merged**